website/docs/plugin/sdkv2/resources/write-only-attributes.mdx

```diff
@@ -0,0 +1,177 @@
+---
+page_title: Resources - Write-only Attributes
+description: Implementing write-only attributes within resources.
+---
+
+# Resources - Write-only Attributes
+
+~> **NOTE:** Write-only attributes are only supported in Terraform `v1.11` or higher
+
+Write-only attributes are managed resource attributes whose values are not saved to the Terraform plan or state artifacts. Write-only attributes
+should be used to handle secret values that do not need to be persisted in Terraform state, such as passwords, API keys, etc.
+The provider is expected to be the terminal point for an ephemeral value,
+which should either use the value by making the appropriate change to the API or ignore the value. Write-only attributes can accept [ephemeral values](/terraform/language/resources/ephemeral).
+
+## General Concepts
+
+The following are high level differences between regular attributes and write-only attributes:
```

Comment:
Perhaps instead of "regular", use language like "configuration" or just directly reference `Required` / `Optional` attributes. "Non-computed" is also a potential option